sql/resource_sql_alerts.go

```diff
@@ -46,14 +47,18 @@ func (a *AlertEntity) toCreateAlertApiObject(s map[string]*schema.Schema, data *
 		Op:            a.Options.Op,
 		Value:         a.Options.Value,
 	}
-
-	return ca, nil
+	// This is a workaround for Go SDK problem, will be fixed there.
+	var err error
+	if a.Options.EmptyResultState != "" {
+		err = ca.Options.EmptyResultState.Set(a.Options.EmptyResultState)
```

Comment:
We should throw an error if the value is not supported

> Currently supported values are `unknown`, `triggered`, `ok`

Reply:
The check should be done on the backend. The main problem right now is that I can't simply do `ca.Options.EmptyResultState.Set(a.Options.EmptyResultState)` if the value is omitted - SDK returns an error.